fdbrpc/FlowTransport.actor.cpp

```diff
@@ -1435,6 +1438,10 @@ NetworkAddress FlowTransport::getLocalAddress() const {
 	return self->localAddresses.address;
 }
 
+std::unordered_map<NetworkAddress, Reference<struct Peer>>* FlowTransport::getAllPeers() {
```

Comment:
Do callers need to modify the contents of this map? I would feel better if this could be made `const`:
```suggestion
const std::unordered_map<NetworkAddress, Reference<struct Peer>>& FlowTransport::getAllPeers() const {
```

Reply:
Good point. Done. The first version does modify peers here, but later I changed my mind.